x-pack/plugins/data_visualizer/public/application/common/util/field_types_utils.ts

```diff
@@ -127,6 +127,9 @@ export function kbnTypeToJobType(field: DataViewField) {
       }
       break;
     case KBN_FIELD_TYPES.NUMBER:
+      if (field.esTypes?.every((d) => d === 'aggregate_metric_double')) {
```

Comment:
Would there be a scenario that there can be multiple numeric es types for one field? E.g. ['double', 'aggregate_metric_double']. If it's exclusive or, we can just check using `.some` here 🤔  But it's also not a blocker, I don't think it's likely that esTypes will have that many types anyway for non-text fields.

Reply:
Sounds good! Looked a bit into how `_field_caps` works and looks like `.some()` makes the most sense here. Ready for another look.